frontend/vite.config.ts

```diff
@@ -1,7 +1,16 @@
-import { defineConfig } from 'vite'
-import react from '@vitejs/plugin-react'
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
 
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
-})
+  server: {
+    proxy: {
+      "/api/v1/questions": {
+        target: "http://localhost:8080",
```

Comment:
Might be more optimal to use `http://question-service:8080` and `http://mongo-express:8081` to keep connections within the Docker network

Reply:
Can't do that this is the front end

Reply:
Unless you build it into a docker container but at that point you'd already build it so the proxy wouldn't be there

Reply:
Ah that's fair I was talking along the lines of when we integrate this into an overarching docker-compose file for Assignment 4